src/lib.rs

```diff
@@ -92,3 +92,57 @@ pub(crate) const LOG_INVALID_SESSION: &str = "Invalid session";
 pub extern "C" fn zc_init_logger() {
     let _ = env_logger::try_init();
 }
+
+#[test]
+#[cfg(feature = "default")]
+fn test_default_features() {
```

Comment:
I believe this test is too strict. E.g., adding a transport to the default set of features in zenoh Rust will make this test fail. However, that kind of addition in the features set should not introduce any failure in zenoh-c.

Reply:
Fixed